dogfood/main.tf

```
@@ -36,7 +36,7 @@ resource "coder_agent" "dev" {
     #!/bin/sh
     set -x
     # install and start code-server
-    curl -fsSL https://code-server.dev/install.sh | sh
+    curl -fsSL https://code-server.dev/install.sh | sh -- --version 4.8.3
```

thank you @code-asher for catching!

```suggestion
    curl -fsSL https://code-server.dev/install.sh | sh -s -- --version 4.8.3
```